src/file.js

```diff
@@ -24,22 +24,8 @@ Sk.builtin.file = function (name, mode, buffering) {
     } else if (this.name === "/dev/stderr") {
         this.fileno = 2;
     } else {
-        if (Sk.inBrowser) {  // todo:  Maybe provide a replaceable function for non-import files
```

Comment:
Oh yeah, I may have been a bit rigorous there, I figured that was an implementation detail that didn't really belong in `file` and should be implemented in a `Sk.read` implementation. I'm glad to put it back. But I'd rather then specify a default `Sk.read` implementation rather then adding these things to `file`.